pom.xml

```diff
@@ -124,7 +124,6 @@
             <configuration>
               <artifactSet>
                 <includes>
-                  <include>net.sourceforge.htmlunit:*</include>
```

Comment:
shaded version was not being used anywhere as far as I can tell so let's not pretend we're doing it anymore